kube/src/api/crds.rs

```diff
@@ -157,6 +161,18 @@ mod test {
         assert_eq!(req.method(), "PATCH");
     }
 
+    #[test]
+    fn raw_resource_in_default_group() {
+        let r: Resource = CustomResource::kind("Service")
+            .group("")
+            .version("v1")
+            .into_resource();
+
+        let pp = PostParams::default();
+        let req = r.create(&pp, vec![]).unwrap();
+        assert_eq!(req.uri(), "/api/v1/services?");
```

Comment:
Seems sensible given what the k8s-openapi impl is: https://docs.rs/k8s-openapi/0.8.0/src/k8s_openapi/v1_18/api/core/v1/service.rs.html#1180-1185